### PR TITLE
remove IProtocolFeeController

### PR DESCRIPTION
**Pull request by snreynolds**
## Related Issue

Which issue does this pull request resolve?

### CODE REVIEW

**Review by hensha256 — APPROVED** [review]

File: test/ProtocolFeesImplementation.t.sol [hunk 1]
```diff
@@ -28,7 +27,7 @@ contract ProtocolFeesTest is Test, GasSnapshot, Deployers {
 
     function setUp() public {
         protocolFees = new ProtocolFeesImplementation();
-        feeController = new ProtocolFeeControllerTest();
+        feeController = makeAddr("feeController");
```
Comment (hensha256) [inline test/ProtocolFeesImplementation.t.sol:30]: definitely a nit but theres now a load of `address(feeController)` all over the tests which could just be `feeController`

**Review by wjmelements — APPROVED** [review]

File: test/ProtocolFeesImplementation.t.sol (hunk 1 above)
Comment (snreynolds) [reply test/ProtocolFeesImplementation.t.sol:30]: good catch

**Review by hensha256 — DISMISSED** [review]

File: src/ProtocolFees.sol [hunk 2]
```diff
@@ -21,12 +20,12 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
...
-    IProtocolFeeController public protocolFeeController;
+    address public protocolFeeController;
 
     constructor() Owned(msg.sender) {}
 
     /// @inheritdoc IProtocolFees
-    function setProtocolFeeController(IProtocolFeeController controller) external onlyOwner {
+    function setProtocolFeeController(address controller) external onlyOwner {
         protocolFeeController = controller;
         emit ProtocolFeeControllerUpdated(address(controller));
```
Comment (hensha256) [inline src/ProtocolFees.sol:30]:
```suggestion
        emit ProtocolFeeControllerUpdated(controller);
```

Comment (hensha256) [reply src/ProtocolFees.sol:30]: and same below in `setProtocolFee` and `collectProtocolFees` -> `address(protocolFeeController)`

**Review by hensha256 — APPROVED** [review]

**Pull request merged**